src/main/webapp/index.html

```diff
@@ -21,7 +21,7 @@ <h1>
   </div>
   <button onclick="graph.generateGraph()">Get Graph</button>
   <br />
-
+  <div id="num-mutation-display"></div>
```

Comment:
I noticed in your demo that there were 2 graph divs. Could you delete one of them?